apps/ereg-batch-status-service/config.yml

```diff
@@ -39,4 +41,4 @@ spec:
       cpu: 500m
       memory: 5000Mi
   ingresses:
-    - "https://ereg-batch-status-service.dev.intern.nav.no"
\ No newline at end of file
+    - "https://ereg-batch-status-service.dev-fss-pub.nais.io"
```

Comment:
Må vi bytte til ny ingress igjen eller er det bare spesielt tilfelle her med jenkins? 😅

Reply:
Nei dette er for å komunisere fra dev-gcp til dev-fss. I den retningen er det ekstra strenge sikkerhetstiltak

Reply:
Åja skjønner 👍